Ys senegal2019

yining-sun opened the pull request:
I followed the same approach that I used in Senegal2018. But it seems like the file cannot be automatically merged. 

Let me know if there is any problems.

yining-sun committed:
Senegal2019

- Run the codes smoothly
- No observations in the intermediate quality control file due to the same reason as the Senegal2018. There is simply no SN2019 in the HEFPI and DHS.dta.

yining-sun committed:
Senegal2019

- Run the codes smoothly
- No observations in the intermediate quality control file due to the same reason as the Senegal2018. There is simply no SN2019 in the HEFPI and DHS.dta.

yining-sun committed:
Senegal2019_ Repush

- Changed the codes in 13_adult.do to make the codes run smoothly
- No quality control observations the as "SN2019" not present in surveyid in HEFPI and DHS.dta

wengxyu1030 commented:
There are conflicts when I try to merge the pull request to the master:
![image](https://user-images.githubusercontent.com/63058112/117171156-a16f6b00-ad98-11eb-83ca-89d0aae62664.png)
 This is due to the branch you worked on is not synced with the master branch. Please use fetch and rebase function in GitHub desktop to avoid this. About rebase you can refer to this guide: https://docs.github.com/en/desktop/contributing-and-collaborating-using-github-desktop/syncing-your-branch#:~:text=Force%20push%20origin.-,Use%20the%20Branch%20drop%2Ddown%20and%20click%20Rebase%20Current%20Branch,command%20line%2C%20or%20another%20tool.

wengxyu1030 commented on DHS_Recode_VII.do:
```diff
@@ -31,16 +31,18 @@ global OUT "${root}/STATA/DATA/SC/FINAL"
     
 	do "${DO}/0_GLOBAL.do"
 	
+	global DHScountries_Recode_VII "Senegal2019"
```
Please do not apply the survey name to global as discussed in the final pull request.

fansi-sifan committed:
resolve conflicts

Merge branch 'master' into YS-Senegal2019

# Conflicts:
#	.DS_Store
#	13_adult.do
#	DHS_Recode_VII.do

fansi-sifan committed:
add DS_Store to gitignore

wengxyu1030 reviewed (commented):
Aline manually resolved the conflicts on her end.
